Bug :테스트 코드에서 Lombok @Slf4j 인식 불가

## Description
src/test/java 경로에 있는 테스트 클래스에서 `@Slf4j` 어노테이션을 사용하면,
IDE(IntelliJ)에서는 정상 동작하지만 gradle clean build 시에는 log 필드를 찾지 못하는 컴파일 에러가 발생합니다.

```
error: cannot find symbol
                log.info("[TEST] Slack Alert skipped: {}, {}", httpServletRequest.getRequestURI(), t.getMessage());
                ^
  symbol: variable log
```

--- a/build.gradle
+++ b/build.gradle
@@ -134,6 +134,8 @@ dependencies {
     developmentOnly 'org.springframework.boot:spring-boot-devtools'
     annotationProcessor 'org.projectlombok:lombok'
     compileOnly 'org.projectlombok:lombok'
+    testCompileOnly 'org.projectlombok:lombok'
+    testAnnotationProcessor 'org.projectlombok:lombok'
 
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'org.springframework.security:spring-security-test'

